.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Basic .gitattributes for a R repo.
+
+# Source files
+# ============
+*.Rdata binary
+*.RData binary
+*.rda   binary
+*.rdb   binary
+*.rds   binary
+*.Rd    text
+*.Rdx   binary
+*.Rmd	  text
+*.R  	  text
+*.Rproj text
+*.[Rr]md   linguist-detectable
+
+R text eol=crlf
```

Comment:
no this line should be on 13

*.R text eol=crlf

Reply:
with the * at the beginning

Reply:
Done.

Reply:
so line 17 should go :)